### DISCUSSION TITLE
Intensities reported in XIC files from --vis command

### DISCUSSION THREAD

**Discussion by sfilip-fn** [vdemichev/DiaNN#505] [Q&A] [answered]
Hi Vadim,

What are the intensities reported in "report.XIC.tsv" file generated by using --vis command? Are these raw or normalized intensities?

Thank you,
Szymon

**Comment by vdemichev** [comment, marked as answer]
Hi Szymon,

These are raw intensitites. That is these chromatograms look exactly as DIA-NN sees them when doing peak scoring.

Best,
Vadim

**Reply by sfilip-fn** [reply]
Hi Vadim

Thank you for clarification!

Best,
Szymon
